src/argilla/server/security/model.py

```diff
@@ -82,6 +83,7 @@ def get(self, key: str, default: Any) -> Any:
 class User(BaseModel):
     """Base user model"""
 
+    id: UUID = Field(default_factory=uuid.uuid4)
```

Comment:
We should not need defaults for these fields because `User` model should be used to represent existent users from database.

For creating users we should use `UserCreate` model and maybe we should move all the validations to that model.